src/rust/Cargo.toml

```diff
@@ -1,30 +1,91 @@
 [workspace]
 resolver = "2"
-members = ["aggregator", "api", "db", "dbv2", "types", "sdk", "sdk/example"]
+members = [
+  "aggregator",
+  "api",
+  "db",
+  "dbv2",
+  "types",
+  "sdk",
+  "sdk/example",
+  "dependencies/aptos-indexer-processors/rust/processor",
+]
 exclude = ["dependencies"]
 
+[workspace.package]
+authors = ["Econia Labs <developers@econialabs.com>"]
+license = "Business Source License"
+homepage = "https://www.econialabs.com/"
+publish = false
+repository = "https://github.com/econia-labs/econia"
+edition = "2021"
+rust-version = "1.70"
+
 [workspace.dependencies]
+aptos-indexer-protos = { path = "dependencies/aptos-indexer-processors/rust/aptos-indexer-protos" }
+server-framework = { path = "dependencies/aptos-indexer-processors/rust/server-framework" }
+aptos-moving-average = { path = "dependencies/aptos-indexer-processors/rust/moving-average" }
+
+anyhow = "1.0.62"
+async-trait = "0.1.53"
 axum = "0.6.19"
-bigdecimal = "0.3.1"
-chrono = "0.4.26"
-diesel = "2.1.0"
+backtrace = "0.3.58"
+base64 = "0.13.0"
+bcs = { git = "https://github.com/aptos-labs/bcs.git", rev = "d31fab9d81748e2594be5cd5cdf845786a30562d" }
+bigdecimal = { version = "0.3.1", features = ["serde"] }
+chrono = { version = "0.4.26", features = ["clock", "serde"] }
+clap = { version = "4.3.5", features = ["derive", "unstable-styles"] }
+diesel = { version = "2.1.0", features = [
+    "chrono",
+    "postgres",
+    "r2d2",
+    "numeric",
+    "serde_json",
+] }
+diesel_migrations = { version = "2.1.0", features = ["postgres"] }
 diesel-derive-enum = "2.1.0"
 dotenvy = "0.15.7"
 envy = "0.4.2"
 field_count = "0.1.1"
+futures = "0.3.24"
 futures-util = "0.3.28"
+gcloud-sdk = { version = "0.20.4", features = [
```

Comment:
these are not perfectly alphabetical because i don't know how to make it sort them